camera/cameraDisplayModule.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import os
+import robomodules as rm
+from messages import message_buffers, MsgType
+
+ADDRESS = os.environ.get("BIND_ADDRESS","localhost")
+PORT = os.environ.get("BIND_PORT", 11297)
+
+FREQUENCY = 10
+
+class CameraDisplayModule(rm.ProtoModule):
+    def __init__(self, addr, port):
+        self.subscriptions = [MsgType.CAMERA_MSG]
+        super().__init__(addr, port, message_buffers, MsgType, FREQUENCY, self.subscriptions)
+        self.value = -1
+        self.sub_ticks = 0
+        self.subbed = True
+
+    def msg_received(self, msg, msg_type):
+        # This gets called whenever any message is received
+        # We received pickled frames here.
+        if msg_type == MsgType.CAMERA_FRAME_MSG:
+            self.frame = msg.cameraFrame
+
+
+    def tick(self):
+        # this function will get called in a loop with FREQUENCY frequency
+        # process the serialized frame
+        displaySerializedImage(self.frame)
```

Comment:
since `self.frame` is a instance variable, you don't need to pass it in.
Since `displaySerializedImage` is an instance method, you should call it with `self.displaySerializedImage`.